js/directives/lucca/user-picker.js

```diff
@@ -188,6 +192,14 @@
 		var init = true; // boolean to initialise the connected user
 		var myId; // used for 'display me first' feature
 
+		/** HttpService **/
+		var httpService = function(method){
+			if($scope.customHttpService &&  $scope.customHttpService[method]){
+				return $scope.customHttpService[method];
+			}
+			return $http[method];
+		};
+
```

Comment:
I don't like the fact that your function is called `httpService` but returns a `httpMethod`